### DISCUSSION TITLE
update-guide/

### DISCUSSION THREAD

**Discussion by giscus** [CommandPost/FCPCafe#269] [Website Discussions]
# update-guide/

All software has bugs, and new updates to software can sometimes have regressions that break features that were working fine in previous versions.

https://fcp.cafe/update-guide/

**Comment by waternyc** [comment]
great stuff. Just my first comment here...hello to all.

**Comment by simplevideoedit** [comment]
So far I am using Sonoma 14.1 and Final Cut Pro 10.6.10 on my iMac Pro and M1 Macbook Air and things have been pretty smooth. A couple instances of crashes but no more than I was usually having. Looking forward to 10.7!

**Comment by muanzo** [comment]
Accentize dxRevive and Other Accentize  plugins  not working after Upgrading to Sonoma 14.3 and FCP 10.7.1.

**Comment by rodrigokirchner** [comment]
Great write up - here the FCP 10.8 bugs:
1. Audio scamming - work around - click transform tool on and off - works again.
2. AUhosting crashes - plugins from izotope and other more advanced plugins crashing after the update - workaround  not using AU plugins...
3. Keyframes / masks in viewer lagging - If clip video animation is open, movement keyframes on transform in the video viewer start lagging until it crashes some background library and the whole system starts slowing down - needs restart.... workaround leave video animations closed....

**Reply by belisoful** [reply]
there are a lot more than that.

**Comment by belisoful** [comment]
🔑🙏🔥👀 
Step 1: Right click on the Final Cut Pro application and select "Compress." 
    This saves a copy of your current version.  I like to put the version number on the file too.  
Step 2: There is no step 2.

You can test upgrade this way, and always be able to revert. 🎯  Just be sure to have backups of your files before you open them and any "version upgrades" are applied to the file.  Those are atypical? though.

**Comment by hexeric** [comment]
**stay away from macOS 15, even from 15.3.1 if you can!**
there are so many problems with sequoia, also concerning video and FCP. sequoia is the messiest release since ever: https://www.reddit.com/r/MacOS/comments/1fihlge/macos_15_sequoia_bugs_and_issues_megathread/?sort=new
my personal experience has been bad too, as have my colleagues'.
